### DISCUSSION TITLE
Is there an easy way to extract the metadata?

### DISCUSSION THREAD

**Discussion by josecelano** [Tribler/tribler#7788] [Q&A]
Hi, I'm contributing to a centralized [Index](https://github.com/torrust/torrust-index). I would like to run some tests with real data. I have done it in the past with other public datasets like [academic torrents](https://academictorrents.com/). I have been looking for a torrent dataset with metadata. I see Tribler uses torrents to store the metadata too and there is an API, but I'm not sure if I could get all torrents and their metadata from all channels using the API (and for sure it's don't going to be a simple task).

Is there an easy way to export that info into an interoperable format like CSV or JSON files?

**Comment by synctext** [comment]
Hi, Just install Tribler, browse the discovered content. Let it discover stuff overnight and then you have lots of torrents to explore on you local harddrive.
No export tool, but the format is simplistic.
